addons/project/models/project_task.py

```diff
@@ -405,6 +405,16 @@ def _search_is_closed(self, operator, value):
             return NotImplemented
         return [('state', 'in', searched_states)]
 
+    def _get_rotting_depends_fields(self):
+        return super()._get_rotting_depends_fields() + ['is_closed']
+
+    def _get_rotting_domain(self):
+        return fields.Domain.AND([
+            super()._get_rotting_domain(), [
+                ('is_closed', '=', False),
+            ],
+        ])
```

Comment:
Much simpler with operators. AND/OR should usually not be used if you manually need to build its list/iterator parameter from otherwise static elements.
```suggestion
    def _get_rotting_domain(self):
        return super()._get_rotting_domain() & Domain('is_closed', '=', False)
```